picard/ui/options/interface.py

```diff
@@ -223,8 +222,8 @@ def _all_list_items(self):
 
     def _added_actions(self):
         actions = self._all_list_items()
-        actions = [action for action in actions if action != 'separator']
-        return set(actions)
+        actions = set(action for action in actions if action != 'separator')
+        return actions
```

Comment:
Make it one line:
`return set(action for action in self._all_list_items() if action != 'separator')`